packages/app/src/app/pages/Dashboard/Components/Header/index.tsx

```diff
@@ -57,10 +57,7 @@ export const Header = ({
         width: `calc(100% - ${2 * GUTTER}px)`,
         maxWidth: GRID_MAX_WIDTH - 2 * GUTTER,
         marginX: 'auto',
-        borderStyle: 'solid',
-        borderWidth: 0,
-        borderBottomWidth: 1,
-        borderColor: 'grays.500',
+        borderStyle: 'none',
```

Comment:
I think we can remove the `borderStyle` because the border is no longer added

Reply:
✅